source/_data/sidebar.yml

```diff
@@ -152,6 +152,7 @@ tutorials:
     introduction-to-tutorials: introduction-to-tutorials.html
     project-setup: project-setup.html
     testing-inputs: testing-inputs.html
+    form-submission: form-submission.html
```

Comment:
you need to ensure that these filenames are not generic enough that they won't conflict with ANY doc that's ever added in the future (hence why I renamed 'introduction').

The idea is that you can always link directly from cypress.on (https://on.cypress.io/form-submission). Just something to keep in mind while you're added pages. 

Also, you should probably become familiar with the [cypress.on](https://github.com/cypress-io/cypress-on) repo. Mostly, this [manifest.yml](https://github.com/cypress-io/cypress-on/blob/develop/data/manifest.yml) file needs to stay up to date with this sidebar.yml file (merge PR to develop, PR dev to prod, it auto deploys). This should be automated, but it isn't.

Reply:
Gotcha, good to know. I'll poke around that repo a bit.

I'll push another commit with this file renamed too.